changelogs/unreleased/gh-9806-http-set-version.md

```diff
@@ -0,0 +1,3 @@
+## feature/lua/http client
```

Comment:
Regarding the commit message: `Needed for #xxxx` should be `Fixes #xxxx` or `Closes #xxxx`. The problem described in the issue is solved, right?

https://github.blog/2013-01-22-closing-issues-via-commit-messages/

Reply:
Fixed